meta-mender-core/classes/mender-setup-grub.inc

```diff
@@ -17,3 +17,5 @@ python() {
     if d.getVar('MENDER_GRUB_STORAGE_DEVICE'):
         bb.fatal("MENDER_GRUB_STORAGE_DEVICE is deprecated. This is now dynamically determined at runtime.")
 }
+
+MENDER_FEATURES_ENABLE += "mender-efi-boot"
```

Comment:
You've added this, but I can't see that you're using it anywhere.

Reply:
`mender-efi-boot` gets used in `initrdscripts` and `initramfs-module-install*` recipes. It gets enabled in both `mender-setup-systemd-boot.inc` and `mender-setup.inc`. This resolves a regression that we experienced when upgrading to `dunfell`, as those recipes has been changed to use an override to be grub-specific.

Reply:
You mean like [this one](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-core/recipes-core/initrdscripts/initramfs-module-install_%25.bbappend#L3)? Ok, I expect this will be added when moving to dunfell then.